robolectric/src/test/java/org/robolectric/shadows/ShadowFileProviderTest.java

```diff
@@ -0,0 +1,34 @@
+package org.robolectric.shadows;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import android.content.Context;
+import android.net.Uri;
+import androidx.core.content.FileProvider;
+import androidx.test.core.app.ApplicationProvider;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import java.io.File;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.annotation.Config;
+
+/** Tests for {@link ShadowFileProvider} */
+@RunWith(AndroidJUnit4.class)
+@Config(shadows = {ShadowFileProvider.class})
+public class ShadowFileProviderTest {
```

Comment:
This test doesn't currently work. I'm getting this error despite having the dependency defined:

```
> No matching variant of androidx.core:core:1.10.1 was found. The consumer was configured to find an API of a library compatible with Java 8, preferably in the form of class files, preferably optimized for standard JVMs, and its dependencies declared externally, as well as attribute 'artifactType' with value 'jar' but:
    - Variant 'releaseVariantReleaseApiPublication' capability androidx.core:core:1.10.1 declares an API of a library, and its dependencies declared externally:
        - Incompatible because this component declares a component, with the library elements 'aar' and the consumer needed a component, preferably in the form of class files

```

Would appreciate any guidance. 🙏🏽